delira/training/base_trainer.py

```diff
@@ -136,8 +152,7 @@ def __init__(self,
         assert isinstance(save_path, str)
         assert isinstance(losses, dict)
         assert isinstance(optimizer_params, dict)
-        assert isinstance(train_metrics, dict)
-        assert isinstance(val_metrics, dict)
+        assert isinstance(metrics, dict)
```

Comment:
Is there somewhere proper type-checking? (this will raise an `AssertionError` instead of a `ValueError`)

Reply:
currently there isn't, but if you want to change this, feel free to do so.

Note: If you change this, I'd prefer to raise a `TypeError` and not a `ValueError`